docs/Sampling-batchaggregator.md

```diff
@@ -0,0 +1,75 @@
+# Sampling Aggregate
+
+
+Description
+-----------
+Sampling a large dataset flowing through this plugin to pull random records. Supports two types of sampling
+i.e, Systematic Sampling and Reservoir Sampling.
+
+
+Use Case
+--------
+The plugin is used when you want to randomly sample a large dataset based on different sampling types.
+
+
+Properties
+----------
+**sampleSize:** The number of records that needs to be sampled from the input records.
+
+**samplePercentage:** The percentage of records that needs to be sampled from the input records. Either of
+'samplePercentage' or 'sampleSize' needs to be mentioned.
+
+**samplingType:** Type of the Sampling algorithm that needs to be used to sample the data.
+For example: Systematic or Reservoir
```

Comment:
Type of the Sampling algorithm that should to be used to sample the data. This can be either Systematic or Reservoir.